package.json

```diff
@@ -72,12 +74,13 @@
     "react": "^16.1.1",
     "react-dom": "^16.1.1",
     "rollup": "0.66.4",
-    "rollup-plugin-babel": "^4.0.3",
     "rollup-plugin-replace": "^2.0.0",
-    "rollup-plugin-uglify": "^6.0.0"
+    "rollup-plugin-typescript2": "^0.18.0",
+    "rollup-plugin-uglify": "^6.0.0",
+    "ts-node": "^7.0.1",
```

Comment:
why exactly we need this?

Reply:
removed, it was needed to run mocha_test runner, but now as we have moved to jest as test runner, so we don't need it.